products/neo4j.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Neo4j
+permalink: /neo4j
+releasePolicyLink: https://neo4j.com/terms/support-terms/
+category: db
+changelogTemplate: https://github.com/neo4j/neo4j/tags/__LATEST__/
+activeSupportColumn: false
+eolColumn: Support Status
+versionCommand: call dbms.components() yield versions  unwind versions as version return  version;
+releaseDateColumn: true
+iconSlug: neo4j
+auto:
+-   git: https://github.com/neo4j/neo4j.git
+releases:
+-   releaseCycle: "5.2"
+    eol: false
+    latest: "5.2"
```

Comment:
Latest versions and release dates are not properly filled. For instance latest for 5.2 cycle is [5.2.0](https://github.com/neo4j/neo4j/releases/tag/5.2.0), 4.4 cycle is [4.4.15](https://github.com/neo4j/neo4j/releases/tag/4.4.15)...

Reply:
I'm working on it :rocket: